apis/database/v1beta1/rdsinstance_types.go

```diff
@@ -106,6 +106,37 @@ type ScalingConfiguration struct {
 	SecondsUntilAutoPause *int `json:"secondsUntilAutoPause,omitempty"`
 }
 
+// S3RestoreBackupConfiguration defines the details of the S3 backup to restore from.
+type S3RestoreBackupConfiguration struct {
+	// BucketName is the name of the S3 bucket containing the backup to restore.
+	BucketName *string `json:"bucketName"`
```

Comment:
ideally, this and below would be referencers, but personally I don't mind this as a followup-PR at this point, since you've had to wait so long.

Here's an example: https://github.com/crossplane/provider-aws/commit/c9a8b38ae93a01cff5dd1ca7367f226d1eafecda
Docs: https://github.com/crossplane/crossplane-tools/pull/35

Reply:
Yeah, let's leave that for a later time - I would like to get this out the door without changes that would require additional testing, etc. Referencers look pretty neat though!